README.md

```diff
@@ -79,28 +79,22 @@ Check our website [planetary.social](https://planetary.social/) for more info!
 
 ### Setup
 
-Requirements: [Homebrew](https://brew.sh/), and Xcode.
+You should be able to Build and Run in Xcode without installing any external tools other than Xcode.
 
-1. Install `rbenv` and add it to your shell: 
+If you need to change a dependency, install the `cocoapods` dependency manager first:
 
-```
-$ brew install rbenv && rbenv init
-```
-
-2. Install ruby v2.7.5
+1. Install gem
```

Comment:
I think this should say "Install cocoapods" since `gem` is the dependency manager in this case and the command assumes it is already installed. 

I liked having the rbenv instructions in here because I have often had issues with the version of ruby that ships with macOS. But it seems like you want to get rid of it. Are you using the macOS system ruby on your new Mac? Does it work without any issues? If so then I would be fine with getting rid of the rbenv/ruby installation instructions.

Reply:
I added them again. I think it's good practice anyway to use rbenv and can be helpful for those new to macOS.